### PR TITLE
change view button theme and style

### PR DESCRIPTION
**Pull request by Himanshunegi0332**
Targets #1685 

### Description
Change view button theme and style in HR analytics section.

### Checklist:
 I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/hr/recruitment/reports.blade.php [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 <div class="chart-container" align="center">
     <div class="bar-chart-container">
-        <h2 align="left">Analytics</h2>
+        <div ="button" align="left">
+         <button class="btn btn-primary text-white">Analytics </button>  </div>
```
Comment (P4NK4J) [inline resources/views/hr/recruitment/reports.blade.php:8]:
```suggestion
        <div="button" align="left">
               <button class="btn btn-primary text-white">Analytics </button>  
         </div>
```

**Review by P4NK4J — CHANGES_REQUESTED** [review]

**Comment by P4NK4J** [comment]
@Himanshunegi0332 It seems like you misunderstood the requirements, the theme and style needs to be changes for `View` button on that page, it should be similar to other buttons on the portal. In this PR you seem to have changed the style of Analytics heading.

**Review by P4NK4J — CHANGES_REQUESTED** [review]
requesting changes, since style is changed for a different component, than required. Please go through issue description for more details.

**Comment by Himanshunegi0332** [comment]
kindly re-review my task.

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**